### PR TITLE
Fixes #15974: document purging of logs

### PR DESCRIPTION
**Pull request by ncharles**
https://issues.rudder.io/issues/15974

### CODE REVIEW

File: src/reference/modules/administration/pages/procedures.adoc [hunk 1]
```diff
@@ -20,26 +20,33 @@ database.
...
 and putting aside old ones.
 
 This is configurable in `/opt/rudder/etc/rudder-web.properties`, by changing the following
 configuration elements:
 
-* `rudder.batch.reportscleaner.archive.TTL`: Set the maximum report age before archival
-* `rudder.batch.reportscleaner.delete.TTL`: Set the maximum report age before deletion
+* `rudder.batch.reportscleaner.archive.TTL`: Set the maximum report age before archival (in days)
+* `rudder.batch.reportscleaner.delete.TTL`: Set the maximum report age before deletion  (in days)
+* `rudder.batch.reportscleaner.compliancelevels.delete.TTL` : Set the maximum compliance age before remova  (in days)
```
Comment (amousset) [inline src/reference/modules/administration/pages/procedures.adoc]: removal

File: src/reference/modules/administration/pages/procedures.adoc [hunk 2]
```diff
@@ -20,26 +20,33 @@ database.
...
+* `rudder.batch.reportsCleaner.deleteLogReport.TTL` : Set the maximum retention of logs reports, (in runs number, using Nx notation, or in minutes)
```
Comment (amousset) [inline src/reference/modules/administration/pages/procedures.adoc]: Nx is ambiguous, could you give an example?

**Comment by ncharles** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
OK, squash merging this PR

**Pull request merged**